packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -45,30 +26,15 @@ const GitLabScopingSearchResults = (props: Props) => {
   const {queryRef, meetingRef} = props
   const query = usePreloadedQuery(
     graphql`
-      query GitLabScopingSearchResultsQuery($teamId: ID!) {
-        ...GitLabScopingSearchResults_query
-      }
-    `,
-    queryRef,
-    {UNSTABLE_renderPolicy: 'full'}
-  )
-
-  const paginationRes = usePaginationFragment<
-    GitLabScopingSearchResultsPaginationQuery,
-    GitLabScopingSearchResults_query$key
-  >(
-    graphql`
-      fragment GitLabScopingSearchResults_query on Query
-        @argumentDefinitions(
-          projectsFirst: {type: "Int", defaultValue: 20}
-          issuesFirst: {type: "Int", defaultValue: 25}
-          projectsAfter: {type: "String"}
-          issuesAfter: {type: "String"}
-          search: {type: "String"}
-          projectIds: {type: "[ID!]", defaultValue: null}
-        )
-        @refetchable(queryName: "GitLabScopingSearchResultsPaginationQuery") {
```

Comment:
In v1, we're now querying a large number of projects and getting their issues. The user can't scroll down to load more issues. If the issue doesn't exist in the initial query, they can use the search query or filter menu. 

In v2, we'll refactor the nested connections so users can scroll down to load more issues: https://github.com/ParabolInc/parabol/issues/6325

For now, I want to update this query in this parent branch so all child branches follow the same pattern.